tests/generic_functions/foobar_as_str/function.json

```diff
@@ -0,0 +1,17 @@
+{
+  "scriptFile": "main.py",
+  "bindings": [
+    {
+      "type": "foobar",
```

Comment:
Add a test for queue trigger, with datatype set to string

Reply:
Done.